portfolio/src/main/webapp/script.js

```diff
@@ -37,11 +37,11 @@ function addRandomQuote() {
  * This function opens the tab we want to access and hides the unnecessary tabs
  */
 function openTab(evt, tabName) {
-    var tabcontent = document.getElementsByClassName("tabcontent");
+    const tabcontent = document.getElementsByClassName("tabcontent");
     for(var i = 0; i <tabcontent.length; i++) {
```

Comment:
Best to use let instead of var

Reply:
I think I just updated this in my latest commit, Dan recommended that I use const here instead!